flask_security/core.py

```diff
@@ -247,7 +247,7 @@ def _request_loader(request):
         data = _security.remember_token_serializer.loads(
             token, max_age=_security.token_max_age)
         user = _security.datastore.find_user(id=data[0])
-        if user and verify_hash(data[1], user.password):
```

Comment:
I think the reason to call `verify_hash` was to make sure that the token is invalidated once a user changes her password. Can you propose any solution that allows users to invalidate the remember token?